rust/protocol/src/storage/traits.rs

```diff
@@ -62,6 +85,7 @@ pub trait PreKeyStore {
     async fn remove_pre_key(&mut self, prekey_id: PreKeyId, ctx: Context) -> Result<()>;
 }
 
+/// Interface for storing signed pre-keys downloaded from a server. (TODO: ???)
```

Comment:
I'll come back with an explanation of pre-keys and signed pre-keys later; for now feel free to leave this generic.

Reply:
Thanks! Done!

Reply:
[marking this unresolved so I'm less likely to forget about it]